contracts/apps/disputable/IDisputable.sol

```diff
@@ -39,4 +39,6 @@ contract IDisputable is ERC165 {
     function supportsInterface(bytes4 _interfaceId) external pure returns (bool) {
         return _interfaceId == DISPUTABLE_INTERFACE_ID || _interfaceId == ERC165_INTERFACE_ID;
     }
+
+    function appId() public view returns (bytes32);
```

Comment:
Somewhat related discussion, but maybe we should create a new interface, e.g. `IAragonApp` to enforce the `kernel()` and `appId()` functions?

That way `IDisputable` can declare that it's always an `IAragonApp`.

Reply:
I like the idea of `IAragonApp`, I'll create an issue for that :)